### PR TITLE
std.algo.cmp.levenshteinandpath docu

### PR DESCRIPTION
**Pull request by burner**
* Params
* Returns

### CODE REVIEW

File: std/algorithm/comparison.d [hunk 1]
```diff
@@ -1159,6 +1159,15 @@ size_t levenshteinDistance(alias equals = "a == b", Range1, Range2)
 Returns the Levenshtein distance and the edit path between $(D s) and
 $(D t).
 
+Params:
+    equals = The binary predicate to compare the elements of the two ranges.
+    s = The original range.
+    t = The transformation target
+
+Returns:
+    The minimal amount of edits to transform s into t and the edits path
```
Comment (quickfur) [inline std/algorithm/comparison.d]: Nitpick: "minimal number of edits" is better.
Also, "the edits path..." sounds awkward, maybe better is "the sequence of edits to effect this transformation".

**Comment by quickfur** [comment]
LGTM, thanks!

**Comment by quickfur** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

File: std/algorithm/comparison.d [hunk 2]
```diff
@@ -1159,6 +1159,15 @@ size_t levenshteinDistance(alias equals = "a == b", Range1, Range2)
 Returns the Levenshtein distance and the edit path between $(D s) and
 $(D t).
 
+Params:
+    equals = The binary predicate to compare the elements of the two ranges.
```
Comment (yazd) [inline std/algorithm/comparison.d]: binary predicate used* to compare

File: std/algorithm/comparison.d [hunk 3]
```diff
@@ -1159,6 +1159,15 @@ size_t levenshteinDistance(alias equals = "a == b", Range1, Range2)
...
+    s = The original range.
+    t = The transformation target
```
Comment (yazd) [inline std/algorithm/comparison.d]: missing full stop

File: std/algorithm/comparison.d [hunk 4]
```diff
@@ -1159,6 +1159,15 @@ size_t levenshteinDistance(alias equals = "a == b", Range1, Range2)
...
+
+Returns:
+    The minimal amount of edits to transform s into t and the sequence of
```
Comment (yazd) [inline std/algorithm/comparison.d]: missing $(D for s and t

File: std/algorithm/comparison.d [hunk 5]
```diff
@@ -1159,6 +1159,15 @@ size_t levenshteinDistance(alias equals = "a == b", Range1, Range2)
...
+    edits to effect this transformation.
```
Comment (yazd) [inline std/algorithm/comparison.d]: I am not sure 'effect' is a good word to use here.

**Pull request merged**